compiler/modules/nback/number.bal

```diff
@@ -12,6 +12,20 @@ final RuntimeFunction floatToIntFunction = {
     attrs: ["nounwind", "readnone", "speculatable", "willreturn"]
 };
 
+final readonly & map<RuntimeFunction> decimalArithmeticFuncs = decimalArithmeticFuncsInit();
+
+function decimalArithmeticFuncsInit() returns readonly & map<RuntimeFunction> {
```

Comment:
Init is not the right name for something that returns a value. Look at what I have done elsewhere in similar cases (probably something beginning with `create`).

Reply:
Changed